common/util.go

```diff
@@ -437,11 +436,8 @@ func CreateHistoryStartWorkflowRequest(
 	firstDecisionTaskBackoffSeconds := backoff.GetBackoffForNextScheduleInSeconds(
 		startRequest.GetCronSchedule(), now, now)
 	delayStartSeconds := startRequest.GetDelayStartSeconds()
-	if delayStartSeconds > 0 && firstDecisionTaskBackoffSeconds > 0 {
-		return nil, ErrDelayStartSeconds
-	}
 	if delayStartSeconds > 0 {
-		firstDecisionTaskBackoffSeconds = delayStartSeconds
+		firstDecisionTaskBackoffSeconds += delayStartSeconds
```

Comment:
Do we add this delay for the very first cron run or for every scheduled cron runs? 

Let's say if the Cron is running every hour and they wanted to start that workflow in 10 minutes expected delays for each cron runs should be
1st run: delayStartSeconds + BackoffForNextScheduleInSeconds
2nd run: 1hr
3rd run: 1hr
...

NOT
1st run: delayStartSeconds + BackoffForNextScheduleInSeconds
2nd run: delayStartSeconds + 1hr
3rd run: delayStartSeconds + 1hr
...

Could someone verify that change here isn't going to cause that?

Reply:
Have you tried running a test locally with the above changes and observed when the workflows kick off? You should have the workflow started time for each run so you can manually check if it's working as expected.

Reply:
Thanks, tested with this and verified that first one is the behavior. Updated the test plan in the PR too